src/cursor/cursor.ts

```diff
@@ -54,8 +107,49 @@ export interface CursorOptions extends CoreCursorOptions {
   numberOfRetries?: number;
 }
 
+/** @public */
+export interface StreamOptions {
```

Comment:
this duplicates `CursorStreamOptions` below

Reply:
Oops, I think this slipped back in during a merge, thanks for catching